### ISSUE TITLE
GPIO_SPEED_HIGH undeclared

### ISSUE THREAD

**Issue by johschmitz** [adamheinrich/cm-makefile-examples#2]
When compiling the `stm32-cube` example I get the following error:

```
  CC      main.o
main.c: In function 'main':
main.c:19:12: error: 'GPIO_SPEED_HIGH' undeclared (first use in this function)
   .Speed = GPIO_SPEED_HIGH,
            ^
```

**Comment by adamheinrich** [comment]
That might be because F4 parts have more options for speed (and/or there is incosistency in naming among Cube libraries). Let me check what values I'm using in a F4 project.

**Comment by adamheinrich** [comment]
Use one of these (code from STM32F4 HAL):

```
#define  GPIO_SPEED_FREQ_LOW         0x00000000U  /*!< IO works at 2 MHz, please refer to the product datasheet */
#define  GPIO_SPEED_FREQ_MEDIUM      0x00000001U  /*!< range 12,5 MHz to 50 MHz, please refer to the product datasheet */
#define  GPIO_SPEED_FREQ_HIGH        0x00000002U  /*!< range 25 MHz to 100 MHz, please refer to the product datasheet  */
#define  GPIO_SPEED_FREQ_VERY_HIGH   0x00000003U  /*!< range 50 MHz to 200 MHz, please refer to the product datasheet  */
```

Maybe it would be better to have `.Speed` unspecified in the example to have the code more portable (edit: fixed in 445b3ac8b069ff81776f450db674e5726db86e92).
